app/models/manager_refresh/inventory/persister.rb

```diff
@@ -12,6 +12,28 @@ def initialize(manager, target)
     initialize_inventory_collections
   end
 
+  def self.has_inventory(options)
+    # create method on class
+    name = options[:association]
+    define_method(name) do
+      collections[name] ||= begin
+        options[:parent] ||= manager
+        collection = ::ManagerRefresh::InventoryCollection.new(options)
+        if options[:builder_params]
+          options[:builder_params].transform_values! do |value|
+            if value.respond_to? :call
+              value.call(self)
+            else
+              value
+            end
+          end
+        end
+        yield(collection, self) if block_given?
+        collection
```

Comment:
yeah, it's to hook into building the collection

```ruby
has_inventory_cloud_manager_vms do |collection, persister|
  collection.parent = persister.manager.parent_manager
  collection.dependency_attributes[: orchestration_stacks] = persister.orchestration_stacks
end
```

But I'm not even sure it'll work now, because I'm not passing the block down...
So, thanks, I'll drop this...